modular_skyrat/modules/implants/code/augments_arms.dm

```diff
@@ -219,6 +219,81 @@
 	obj_flags |= EMAGGED
 	return TRUE
 
+// Razorwire implant, long reach whip made of extremely thin wire, ouch!
+
+/obj/item/melee/razorwire
+	name = "implanted razorwire"
+	desc = "A long length of cutting wire so impossibly thin that it causes grevious wounds in anything you slash with it. \
+		Its long enough that you'd probably be able to hit someone with it from a little further away than normal."
+	icon = 'modular_skyrat/modules/implants/icons/implants.dmi'
+	icon_state = "razorwire_weapon"
+	righthand_file = 'modular_skyrat/modules/implants/icons/inhands/lefthand.dmi'
+	lefthand_file = 'modular_skyrat/modules/implants/icons/inhands/righthand.dmi'
+	inhand_icon_state = "razorwire"
+	w_class = WEIGHT_CLASS_BULKY
+	sharpness = SHARP_EDGED
+	force = 18
+	demolition_mod = 0.25 // This thing sucks at destroying stuff
+	wound_bonus = 10
+	bare_wound_bonus = 20
+	weak_against_armour = TRUE
+	reach = 2
+	hitsound = 'sound/weapons/whip.ogg'
+	attack_verb_continuous = list("slashes", "whips", "lashes")
+	attack_verb_simple = list("slash", "whip", "lash")
+
+/obj/item/organ/internal/cyberimp/arm/razorwire
+	name = "razorwire spool implant"
+	desc = "An integrated spool of razorwire, capable of being used as a weapon when whipped at your foes."
+	items_to_create = list(/obj/item/melee/razorwire)
+	icon = 'modular_skyrat/modules/implants/icons/implants.dmi'
+	icon_state = "razorwire"
+
+// Shell launch system, an arm mounted single-shot shotgun/.980 grenade launcher that comes out of your arm
+
+/obj/item/gun/ballistic/shotgun/shell_launcher
+	name = "shell launch system"
+	desc = "An arm mounted cannon for firing shotgun shells or Tydhouer grenades. You can use a <b>wrench</b> to adjust it for a different ammo type."
```

Comment:
desc = "A mounted cannon seated comfortably in a forearm compartment. This humanitarian device is capable in normal mode of firing essentially any shotgun shell, and can be <b>wrenched</b> to a .980 Tydhouer grenade mode, shells famously seen in the 'Kiboko' launcher."